homeassistant/components/screenlogic/strings.json

```diff
@@ -70,10 +77,151 @@
       "description": "Stops super chlorination.",
       "fields": {
         "config_entry": {
-          "name": "Config Entry",
-          "description": "The config entry to use for this action."
+          "name": "[%key:component::screenlogic::common::service_config_entry_name%]",
+          "description": "[%key:component::screenlogic::common::service_config_entry_description%]"
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "active_alert": {
+        "name": "Active Alert"
+      },
+      "circuit_delay": {
+        "name": "{circuit} Delay"
+      },
+      "freeze_mode": {
+        "name": "Freeze Mode"
+      },
+      "flow_alarm": {
+        "name": "Flow Alarm"
+      },
+      "chem_high_alarm": {
+        "name": "{chem} HIGH Alarm"
+      },
+      "chem_low_alarm": {
+        "name": "{chem} LOW Alarm"
+      },
+      "chem_supply_alarm": {
+        "name": "{chem} Supply Alarm"
+      },
+      "probe_fault_alarm": {
+        "name": "Probe Fault"
+      },
+      "chem_limit": {
+        "name": "{chem} Dose Limit Reached"
+      },
+      "ph_lockout": {
+        "name": "pH Lockout"
+      },
+      "corosive": {
+        "name": "SI Corrosive"
+      },
+      "scaling": {
+        "name": "SI Scaling"
+      },
+      "scg_state": {
+        "name": "Chlorinator"
+      }
+    },
+    "climate": {
+      "body_0": {
+        "name": "Pool Heat",
+        "state_attributes": {
+          "preset_mode": {
+            "state": {
+              "solar": "[%key:component::screenlogic::common::climate_preset_solar%]",
+              "solar_prefered": "[%key:component::screenlogic::common::climate_preset_solar_prefered%]",
+              "heater": "[%key:component::screenlogic::common::climate_preset_heater%]"
+            }
+          }
+        }
+      },
+      "body_1": {
+        "name": "Spa Heat",
+        "state_attributes": {
+          "preset_mode": {
+            "state": {
+              "solar": "[%key:component::screenlogic::common::climate_preset_solar%]",
+              "solar_prefered": "[%key:component::screenlogic::common::climate_preset_solar_prefered%]",
+              "heater": "[%key:component::screenlogic::common::climate_preset_heater%]"
+            }
+          }
+        }
+      }
+    },
+    "number": {
+      "calcium_hardness": {
+        "name": "Calcium Hardness"
+      },
+      "cya": {
+        "name": "Cyanuric Acid"
+      },
+      "total_alkalinity": {
+        "name": "Total Alkalinity"
+      },
+      "salt_tds_ppm": {
```

Comment:
FYI, the names need to be Sentence case

Reply:
Ah, ok. And it's ok that this would be a change from the current Title Case?

Reply:
Yep!

Reply:
So, the user-selected names that come from the device are all Title-Case. Do I need to filter all of those down to Sentence-case?

Reply:
Some user-selected names for circuits returned by the pool controller are proper nouns so rather than try to account for all of them across the different controller types, I'm going to leave them in the case they are sent from the device as. Everything defined in HA I will make Sentence case.